### DISCUSSION TITLE
Explain Execution Plan shows only 4 columns

### DISCUSSION THREAD

**Discussion by m3c-ode** [dbeaver/dbeaver#35903] [Q&A] [answered]
### Description

Trying this feature, I do not get options to choose some other valuable information (time...) such as displayed on the wiki

My view:
![image](https://github.com/user-attachments/assets/2435333e-3b4a-452c-96da-430862655769)

Wiki page: 
https://dbeaver.com/docs/dbeaver/Query-Execution-Plan/#simple-plan-view

Not sure why.

### DBeaver Version

community  24.1.5.202408181605

### Operating System

Windows 

### Database and driver

MySql

**Comment by E1izabeth** [comment, marked as answer]
Hi, @m3c-ode 
Different databases have different number of columns in Execution plan. Mysql has the ones you observe.

**Reply by m3c-ode** [reply]
Sorry for being so late, I had moved on to use `EXPLAIN ANALYZE` instead.
But how come MySQL only uses those columns? How can we confirm this?
